### ISSUE TITLE
Similar projet

### ISSUE THREAD

**Issue by tst2005** [pygy/pack.lua#1]
Hello,

I make a lot of stuff about lua...
I discovered some of your projets like require.lua and LuLPeg.
I'm really happy by your modules !

I see this repository (pack.lua). It seems empty but I understand the goal.
I made [lua-aio](https://github.com/tst2005/lua-aio) that seems similar.

I hope it should help you.
Regards,

**Comment by pygy** [comment]
Yup, this was meant to hold the code I use to build LuLPeg, but I never got around to do it...

I'll have a look at lua-aio.

Leaving this open for now, since your repo is far more useful to whoever may come by here.
